src/Keboola/StorageApi/Components.php

```diff
@@ -145,6 +145,19 @@ public function createConfigurationFromVersion($componentId, $configurationId, $
         );
     }
 
+    public function getConfigurationRow(ListConfigurationRowsOptions $options = null)
```

Comment:
To je divný, ne? Že jsou ty options nepovinný. Minimálně rowId by tam být asi mělo, ne? Myslel jsem, že jsme se dohodli včera, že dává smysl to udělat podle [getConfiguration()](https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/Components.php#L93) a [listComponentConfigurations()](https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/Components.php#L111). Tj. že "list" používá Options a "get" přímo ty jednotlivé parametry.

Reply:
Mas pravdu, uplne jsem to vytesnil a pak me ty list* metody opet zmatly.